src/services/requests/home/index.ts

```diff
@@ -477,9 +477,13 @@ const homeKfiPriceCall = async (): Promise<
     });
 
     if (!res.error || res.error === '') {
-      const data = res?.data?.prices?.Exchanges.find(
-        (exchange: any) => exchange.ExchangeName === 'Klever',
-      );
+      const data =
+        res?.data?.prices?.Exchanges.find(
+          (exchange: any) => exchange.ExchangeName === 'Klever',
```

Comment:
dont need 2 loops here just use the checks together exchange.ExchangeName === 'Klever' || !!exchange.ExchangeName